### PR TITLE
Magento 2.4.2 Beta Release Note minor update December 21

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request (PR) updates the Magento 2.4.2 Beta Release Notes with notes about Composer 2.x compatibility and split db deprecation.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/release-notes/commerce-2-4-2.html
https://devdocs.magento.com/guides/v2.4/release-notes/open-source-2-4-2.html

whatsnew
Updated Magento 2.4.2 Beta release notes.

### CODE REVIEW

**Review by jeff-matthews — APPROVED** [review]

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
I've been asked to remove the split DB instructions, so you'll need to remove the link.

**Review by jeff-matthews — APPROVED** [review]
We're not removing the instructions, so the link is still good.

File: src/guides/v2.4/release-notes/commerce-2-4-2.md [hunk 1]
```diff
@@ -62,7 +62,11 @@ This release contains enhancements to core quality, which improve the quality of
...
+*  Magento 2.4.2 is now compatible with **Composer 2.x**. We recommend that merchants migrate to Composer 2.x. Although you can install this release using Composer 1.x, Composer 1.x will soon reach end-of-life. For an overview of Composer 2.x features, see [Composer 2.0 is now available!](https://blog.packagist.com/composer-2-0-is-now-available/)
+
+The ability to configure a Magento installation to use a split database has been deprecated in this release. Merchants who currently use split database should start planning to revert to or migrate to a single database or use an alternative approach. See the [Deprecation of split database functionality in Magento Commerce](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-split-database-functionality-in-Magento-Commerce/ba-p/465187) DevBlog post for an overview of this issue.  See [Revert from a split database to a single database]({{ page.baseurl }}/config-guide/revert-split-database.html) for migration instructions.
```
Comment (keharper) [inline src/guides/v2.4/release-notes/commerce-2-4-2.md:67]:
```suggestion
*  The ability to configure a Magento installation to use a split database has been deprecated in this release. Merchants who currently use split database should start planning to revert to or migrate to a single database or use an alternative approach. See the [Deprecation of split database functionality in Magento Commerce](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-split-database-functionality-in-Magento-Commerce/ba-p/465187) DevBlog post for an overview of this issue.  See [Revert from a split database to a single database]({{ page.baseurl }}/config-guide/revert-split-database.html) for migration instructions.
```

File: src/guides/v2.4/release-notes/open-source-2-4-2.md [hunk 2]
```diff
@@ -62,7 +62,9 @@ This release contains enhancements to core quality, which improve the quality of
...
+*  Magento 2.4.2 is now compatible with **Composer 2.x**. We recommend that merchants migrate to Composer 2.x. Although you can install this release using Composer 1.x, Composer 1.x will soon reach end-of-life. For an overview of Composer 2.x features, see [Composer 2.0 is now available!](https://blog.packagist.com/composer-2-0-is-now-available/)
+
+The ability to configure a Magento installation to use a split database has been deprecated in this release. Merchants who currently use split database should start planning to revert to or migrate to a single database or use an alternative approach. See the [Deprecation of split database functionality in Magento Commerce](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-split-database-functionality-in-Magento-Commerce/ba-p/465187) DevBlog post for an overview of this issue.  See [Revert from a split database to a single database]({{ page.baseurl }}/config-guide/revert-split-database.html) for migration instructions.
```
Comment (keharper) [inline src/guides/v2.4/release-notes/open-source-2-4-2.md:67]:
```suggestion
*  The ability to configure a Magento installation to use a split database has been deprecated in this release. Merchants who currently use split database should start planning to revert to or migrate to a single database or use an alternative approach. See the [Deprecation of split database functionality in Magento Commerce](https://community.magento.com/t5/Magento-DevBlog/Deprecation-of-split-database-functionality-in-Magento-Commerce/ba-p/465187) DevBlog post for an overview of this issue.  See [Revert from a split database to a single database]({{ page.baseurl }}/config-guide/revert-split-database.html) for migration instructions.
```

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**